lib/tasks/config_files.rake

```diff
@@ -78,7 +78,7 @@ namespace :config_files do
               'VCSPATH=alaveteli ' \
               'SITE=alaveteli ' \
               'SCRIPT_FILE=config/alert-tracks-debian.example ' \
-              'RUBY_VERSION=1.9.1 '
+              'RUBY_VERSION=2.1.5 '
```

Comment:
Why 2.1.5? I assume because this is the default included with Debian Jessie – our current default Vagrant VM?

Commit message should explain this.

Reply:
Good point - thanks. 2.1.5 is the most commonly available Ruby at this point in time (off the top of my head - will check for the purposes of the commit message - Stretch adds 2.3.x)